stl/inc/vector

```diff
@@ -3203,7 +3203,7 @@ public:
         }
 
         // max_size bound by underlying storage limits
-        return _Ints_max * _VBITS;
+        return static_cast<size_type>(_Ints_max * _VBITS);

```

Comment:
No change quested. This is related to #5546, and I've found several other places where `static_cast<size_type>()` seems necessary. We can complete the cleanup in another PR.